.github/workflows/code_security_scan.yml

```diff
@@ -0,0 +1,28 @@
+name: Code Security Scan
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+concurrency:
+  group: code-security-scan-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs: 
+  code_security_scan:
+    if: github.event.pull_request.draft == false
+    uses: XanaduAI/cloud-actions/.github/workflows/static_code_vulnerability_analysis.yml@sc-86314-sec-scanning
```

Comment:
This is something we need to maintain for the CI here, or something separate across the org?

Reply:
That reference will get updated to `@main`.

I can put it here but the goal was to not duplicate the code logic across pl/lightning/catalyst. We have been using cloud-actions repo for this purpose for a while now

Reply:
All good so, thanks.